src/app/exp-feature-page/exp-feature-page.component.less

```diff
@@ -0,0 +1,5 @@
+svg {
```

Comment:
I could add a class, it was easiest to just point to the `circle` element. But that is an easy fix.